src/libs/Navigation/CustomActions.js

```diff
@@ -17,13 +18,13 @@ function pushDrawerRoute(screenName, params, navigationRef) {
 
 
         // Avoid the navigation and refocus the report if we're trying to navigate to our active report
+        // We use our RootState as the dispatch's state is relative to the active navigator and might
+        // not contain our active report.
         const rootState = navigationRef.current.getRootState();
-        const activeReportID = rootState.routes[0].state.routes[0].params.reportID;
+        const activeReportID = lodashGet(rootState, 'routes[0].state.routes[0].params.reportID', '')
```

Comment:
This doesn't look very good but should be safe. Maybe a better idea would be making this a `getActiveReport(rootState)` function that searches for a route of `type === 'drawer'` and get's the reportID from it.

Not sure.